pkg/cnb/project_descriptor_test.go

```diff
@@ -18,16 +20,20 @@ func TestProcessProjectDescriptor(t *testing.T) {
 }
 
 func testProcessProjectDescriptor(t *testing.T, when spec.G, it spec.S) {
+	var buf *bytes.Buffer
+	var logger *log.Logger
 	var (
 		appDir, platformDir, projectToml string
 	)
 
 	it.Before(func() {
 		var err error
+		buf = new(bytes.Buffer)
+		logger = log.New(buf, "", 0)
 		appDir, err = ioutil.TempDir("", "appDir")
-		require.NoError(t, err)
```

Comment:
I think `NoError` reads better but it's not a big deal

Reply:
makes sense